base_transaction_id/views/invoice.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <record model="ir.ui.view" id="invoice_view_custom">
+    <field name="name">customer.invoice.transaction.inherit</field>
+    <field name="model">account.invoice</field>
+    <field name="inherit_id" ref="account.invoice_form"/>
+    <field name="arch" type="xml">
+      <xpath expr="//page[@name='other_info']/group/group/field[@name='origin']" position="after">
```

Comment:
I'd replace `group/group` with `//` to make it more robust in case is moved. Even if as you are selecting the field origin.... why not selecting only that? Very likely we don't have the same filed twice. No?

Reply:
Yes, can obviously be simplified.

It was probably failing in v10 as it was changed from
`<field name="origin" position="after">` to this.

If there are 1 origin field in the invoice lines before the origin field of the main object.

So it might be safer to stick with the expr but simplify it a bit.